cvat-ui/src/components/annotation-page/review/issues-aggregator.tsx

```diff
@@ -74,19 +78,16 @@ export default function IssueAggregatorComponent(): JSX.Element | null {
         if (canvasReady) {
             const { geometry: updatedGeometry } = canvasInstance;
             setGeometry(updatedGeometry);
+            const listener = geometryListener.bind(null, setGeometry, canvasInstance);
 
-            const geometryListener = (): void => {
-                setGeometry(canvasInstance.geometry);
-            };
-
-            canvasInstance.html().addEventListener('canvas.zoom', geometryListener);
-            canvasInstance.html().addEventListener('canvas.fit', geometryListener);
-            canvasInstance.html().addEventListener('canvas.reshape', geometryListener);
+            canvasInstance.html().addEventListener('canvas.zoom', listener);
+            canvasInstance.html().addEventListener('canvas.fit', listener);
+            canvasInstance.html().addEventListener('canvas.reshape', listener);
 
             return () => {
-                canvasInstance.html().removeEventListener('canvas.zoom', geometryListener);
-                canvasInstance.html().removeEventListener('canvas.fit', geometryListener);
-                canvasInstance.html().addEventListener('canvas.reshape', geometryListener);
```

Comment:
Probably it was the mistake. We need ``removeEventListener`` here. No need to change other things